freqtrade/plugins/pairlist/RemotePairList.py

```diff
@@ -142,7 +142,10 @@ def gen_pairlist(self, tickers: Tickers) -> List[str]:
         """
 
         if self._init_done:
-            pairlist = self._pair_cache.get('pairlist')
+            if self._pair_cache.get('pairlist') == ["Empty"]:
```

Comment:
when a empty pairlist is returned for example: {"pairs":[],"refresh_period":1800} , it will keep fetching the pairlist from the url every bot_loop_seconds (default 5) due to the cache being empty

if it is set to "None" the cache wont reset, i need a cache item for a empty pairlist to still use the caching functionality, i can change it to List[None], that would work, if thats any better?